<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_his_qushibiao</class>
 <widget class="QDialog" name="Dlg_his_qushibiao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>趋势表</string>
  </property>
  <widget class="QPushButton" name="m_btn_back_fast">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_interval">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_end">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_return">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_fast">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>435</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间间隔</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_end">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv_para">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
